druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/server/lookup/namespace/RocksDBManager.java

```diff
@@ -21,6 +21,7 @@
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
 import org.apache.hadoop.hdfs.DistributedFileSystem;
+import org.eclipse.jetty.util.log.Log;
```

Comment:
Is that the right logger?

Reply:
mistakenly added by auto import. removed it